tests/test_units/test_dunder_methods.py

```diff
@@ -0,0 +1,80 @@
+"""Explicitly test all dunder methods."""
+
+from fmu.dataio._model.fields import Parameters
+from fmu.dataio._model.global_configuration import Stratigraphy, StratigraphyElement
+
+# --------------------------------------------------------------------------------------
+# Parameters
+# --------------------------------------------------------------------------------------
+
+testset_stratigraphy = Stratigraphy(
```

Comment:
You could consider implementing this as a fixture in pytest. Then you can take the fixture as input to your test. Example here:
https://github.com/equinor/fmu-dataio/blob/b30c170d7d76cab61d4b3eea86f7a1fb4ba9ca7f/tests/test_units/test_dictionary.py#L15 

More info here https://www.geeksforgeeks.org/fixtures-in-pytest/